airbyte-cdk/python/airbyte_cdk/sources/file_based/file_based_stream_reader.py

```diff
@@ -30,11 +29,9 @@ def open_file(self, file: RemoteFile) -> IOBase:
     def get_matching_files(
         self,
         globs: List[str],
-        from_date: Optional[datetime] = None,
```

Comment:
@clnoll let me know if you disagree with this change. I figured it's easier to add a parameter we need than to remove one later on

Reply:
Makes sense to me!